index.html

```diff
@@ -34,7 +34,7 @@
   <span class="attr">pattern</span>: <span class="string">`# !`</span>,
   <span class="attr">formatWithSymbol</span>: <span class="literal">true</span>
 }).format(); <span class="comment">// => "1.23 $"</span></code></pre><h3 id="negativepattern-default-">negativePattern <em>default</em>: <code>-!#</code></h3><p>Allows you to customize the negative format pattern using <code>!</code> as replacement for the currency symbol and <code>#</code> as replacement for the currency amount.</p><pre><code class="language-js">currency(<span class="number">-1.23</span>, {
-  <span class="attr">pattern</span>: <span class="string">`(!#)`</span>,
```

Comment:
The documentation is actually built in markdown under [docs/02-options.md](../../tree/master/docs/02-options.md) so this change would get overwritten when it's automatically deployed. If you want to update the option there, I'd be happy to pull in.

Reply:
Thanks, I'll submit a new PR in a sec!